theme/_includes/scripts.html

```diff
@@ -24,8 +24,65 @@
 </script>
 
 <!-- use Bootstrap table formatting for Markdown tables-->
-<script type"text/javascript">
+
+<script type="text/javascript">
   $(document).ready(function () {
     $("table").addClass("table table-striped table-bordered");
   });
 </script>
+
+<!-- list.js for homepage search/filter -->
+<script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/list.js/1.5.0/list.min.js"></script>
+
+<!-- code for filters-->
+<script type="text/javascript">
+var options = {
+  valueNames: [ 'category', 'card-title', 'card-text', 'card-footer', 'sort-title']
+};
+var documentsList = new List ('docs-list', options);
+documentsList.sort('sort-title', { order: "asc" });
+$('#filter-none').click(function() {
+  documentsList.filter();
+  return false;
+});
+$('#filter-1').click(function() {
+  documentsList.filter(function(item) {
+    if (item.values().category == "collection development and management") {
+      return true;
+    } else {
+      return false;
+    }
+  });
+  return false;
+});
+$('#filter-2').click(function() {
+  documentsList.filter(function(item) {
+    if (item.values().category == "preservation") {
+      return true;
+    } else {
+      return false;
+    }
+  });
+  return false;
+});
+$('#filter-3').click(function() {
+  documentsList.filter(function(item) {
+    if (item.values().category == "arrangement and description") {
+      return true;
+    } else {
+      return false;
+    }
+  });
+  return false;
+});
+$('#filter-4').click(function() {
+  documentsList.filter(function(item) {
+    if (item.values().category == "reference and outreach") {
+      return true;
+    } else {
+      return false;
+    }
+  });
+  return false;
```

Comment:
I think lines 48-82 could be refactored significantly as follows:
- List for a click on a common class for all of these button
- Get the data about which category was clicked on by reading a data attribute or something else in the HTML of that button and set that as a variable
- use that variable to do the filtering on the categories